src/osip-core/src/test/java/edu/kit/pse/osip/core/io/files/ClientSettingsWrapperTest.java

```diff
@@ -0,0 +1,180 @@
+    @Test
+    public void testSetServerHostname() {
+        wrapper.setServerHostname(TankSelector.MAGENTA, "123.123");        
+        assertEquals(wrapper.getHostname(TankSelector.MAGENTA), "123.123");
+    }
+    
+    /**
+     * Test setting underflow alarm status
+     */
+    @Test
+    public void testSetUnderflowAlarm() {
+        wrapper.setUnderflowAlarm(TankSelector.YELLOW, true);        
+        assertEquals(wrapper.getUnderflowAlarm(TankSelector.YELLOW), true);
+    }
+    
+    /**
+     * Test setting fill level diagram status
+     */
+    @Test
+    public void testSetFillLevelDiagram() {
+        wrapper.setFillLevelDiagram(TankSelector.YELLOW, true);        
+        assertEquals(wrapper.getFillLevelDiagram(TankSelector.YELLOW), true);
+    }
+    
+    /**
+     * Test getting fetchinterval
+     */
+    @Test
+    public void testGetFetchInterval() {
+        int result = wrapper.getFetchInterval();
+        assertEquals(result, 80);
+    }
+    
+    /**
+     * Test getting overflow alarm status
+     */
+    @Test
+    public void testGetOverflowAlarm() {
+        boolean result = wrapper.getOverflowAlarm(TankSelector.YELLOW);
+        assertEquals(result, false);
+    }
+    
+    /**
+     * Test getting underflow alarm status
+     */
+    @Test
+    public void testGetUnderflowAlarm() {
+        boolean result = wrapper.getUnderflowAlarm(TankSelector.YELLOW);
+        assertEquals(result, false);
+    }
+    
+    /**
+     * Test getting temp diagram status
+     */
+    @Test
+    public void testGetTempDiagram() {
+        boolean result = wrapper.getTempDiagram(TankSelector.YELLOW);
+        assertEquals(result, false);
+    }
+    
+    /**
+     * Test getting fill level diagram status
+     */
+    @Test
+    public void testGetFillLevelDiagram() {
+        boolean result = wrapper.getFillLevelDiagram(TankSelector.YELLOW);
+        assertEquals(result, false);
+    }
+    
+    /**
+     * Test getting hostname
+     */
+    @Test
+    public void testGetHostname() {
+        String result = wrapper.getHostname(TankSelector.MAGENTA);
+        assertEquals(result, "hostNAME");
+    }
+    
+    /**
+     * Test getting port
+     */
+    @Test
+    public void testGetPort() {
+        int result = wrapper.getPort(TankSelector.MAGENTA);
+        assertEquals(result, 4242);
+    }
+    
+    /**
+     * Test saving to file
+     * @throws FileNotFoundException Exception in file param
+     * @throws IOException Exception in IO stream
+     */
+    @Test
+    public void testSaveSettings() throws FileNotFoundException, IOException {
+        wrapper.setServerPort(TankSelector.MAGENTA, 424299);
+        wrapper.saveSettings();
+        ClientSettingsWrapper helpWrapper = new ClientSettingsWrapper(propertiesFile);
+        int helpPort = helpWrapper.getPort(TankSelector.MAGENTA);
+        wrapper.setServerPort(TankSelector.MAGENTA, 4242);
+        wrapper.saveSettings();
+        assertEquals(helpPort, 424299);
```

Comment:
Die Überlegung war den Wert sofort und nicht im after zurückzusetzen, da das nur in dem einen Test erforderlich ist.